CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+
+## [1.14.1] - 2023-07-20
```

Comment:
Please update the footer.
* add link of diff between 1.14.0 and 1.14.1
* update link of unreleased to show diff between 1.14.1 and HEAD

Reply:
Sorry, I add link of diff it.